apps/200-flexdashboard-render-text/index.Rmd

```diff
@@ -35,19 +35,7 @@ Sidebar Content: `r renderText(getmonth())`
 ```{r}
 shinyjster::shinyjster_server(input, output, session)
 shinyjster::shinyjster_js("
-  var jst = jster();
-  jst.add(Jster.shiny.waitUntilStable);
-  jst.add(function() {
-    // make sure on first page
-    $('a[href=\"#section-p1\"]').click();
-  });
-  jst.add(Jster.shiny.waitUntilStable);
-  jst.add(Jster.shiny.waitUntilStable);
-  jst.add(Jster.shiny.waitUntilStable);
-  jst.add(Jster.shiny.waitUntilStable);
-  jst.add(Jster.shiny.waitUntilStable);
-  jst.add(Jster.shiny.waitUntilStable);
-
+  var jst = jster(1);
   function assert_equal(id, txt) {
     assert_equal_qs('#section-' + id, txt);
```

Comment:
This is a such a handy/helpful function. Saved a ton of code repetition across the app. self note: "qs" for query string